Traceback (most recent call last):
  File "/home/death/dev/whimsy/whimsy/infrastructure/__init__.py", line 77, in alrm
    raise RuntimeError('froze!')
RuntimeError: froze!

Traceback (most recent call last):
  File "/home/death/dev/whimsy/whimsy/infrastructure/__init__.py", line 77, in alrm
    raise RuntimeError('froze!')
RuntimeError: froze!

Traceback (most recent call last):
  File "/home/death/dev/whimsy/whimsy/infrastructure/__init__.py", line 77, in alrm
    raise RuntimeError('froze!')
RuntimeError: froze!

